src/main/java/innovimax/mixthem/arguments/Rule.java

```diff
@@ -10,12 +10,14 @@
 * @version 1.0
 */
 public enum Rule { 
-    FILE_1("1", "1", "will output file1", true, 
+    FILE_1("1", "1", "will output first file", true, 
                 EnumSet.noneOf(RuleParam.class), EnumSet.of(FileMode.CHAR, FileMode.BYTE)),
```

Comment:
Remove FILE_1 and FILE_2 since they are sepcial case of FILE_K

Reply:
ok